src/partials/sidebar/constants.js

```diff
@@ -0,0 +1,19 @@
+export const SECTION_CATEGORIES = [
```

Comment:
Nit: these two could probably be combined?

Reply:
good idea